test/packaging/ansible/test.yml

```diff
@@ -15,6 +15,9 @@
 - name: agent installation via newrelic-cli
   import_playbook: installation-newrelic-cli.yml
 
+- name: installation windows
```

Comment:
Should we comment this test until fluent-bit 2.0 is released for Windows?

Reply:
(log-forwarder.yml for Windows)

Reply:
I don't think so. This test tests the installation on windows w/o taking into account the FF or the logging binaries. It asserts the current scenario. Then in the PR with windows it will assert in the new one.

Reply:
Great! Then let's move on 💪